src/Sushi.Mediakiwi.Demonstration/appsettings.json

```diff
@@ -41,7 +41,9 @@
     "disable_caching": false,
     //"local_file_path": "https://localhost/Sushi.Mediakiwi.Files/",
     "file_version": null,
-    "azure_image_container": "testing-mediakiwi"
+    "azure_image_container": "testing-mediakiwi",
+    "sql_install_enabled": true,
+    "sql_install_actions_enabled": true
```

Comment:
@marcmolenwijk Shouldn't these be set to False by default ?
if i understood correctly, setting them to True breaks current development

Reply:
Only the second one... i will set that to false